.github/workflows/docs.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.12'
+          python-version: '3.13'
```

Comment:
Maybe you can update Ubuntu version to 24.04 ?

We're working to update everything we can related to doc generation in the djangoproject.com (see https://github.com/django/djangoproject.com/issues/1634)

Reply:
That's not the case. We have to wait for releasing a new version in https://github.com/actions/python-versions.

Reply:
Also, `docs` OS have to match `djangoproject.com`.

Reply:
We're working on that